packages/esm-form-entry-app/translations/fr.json

```diff
@@ -0,0 +1,53 @@
+{
+  "requiredField": "Ce champs est requis!",
```

Comment:
Could we ensure the ordering of keys is consistent across the three language files? Makes it a little easier to review.